index.css

```diff
@@ -18,6 +18,365 @@
   font-size: 16px;
 }
 
+/* --------- Desktop ---------  */
+
+@media screen and (min-width: 480px) {
+  body {
+    background-color: var(--light-background);
+  }
```

Comment:
- [ ] Dear @shayan1234554321, Good job using a media query for your CSS, But according to the main requirements the value for `min-width` property should be in the range of `768px` or wider. Please note `any value between 768 and 1440 is` valid.